src/translations/de.json

```diff
@@ -152,28 +152,28 @@
     "duplicateWithoutSaving": "Dupliziere ohne Änderungen zu speichern",
     "editLabel": "{{label}} bearbeiten",
     "editing": "Bearbeite",
-    "email": "Email",
-    "emailAddress": "Email Address",
-    "enterAValue": "Geben Sie einen Wert ein",
+    "email": "E-Mail",
+    "emailAddress": "E-Mail-Adresse",
+    "enterAValue": "Gib einen Wert ein",
     "filter": "Filter",
-    "filterWhere": "Filtern Sie {{label}} wo",
+    "filterWhere": "Filter {{label}} wo",
     "filters": "Filter",
-    "globals": "Globalen",
+    "globals": "Globale Dokumente",
```

Comment:
You said 'global documents' is preffered, but I don't think pluralizing the document part is correct as in the case of globals it's a single document. Playing with Google translate makes me think this would be better as 'Globales Dokument'.
Does this make sense?

Reply:
'global' would be translated to 'Globales Dokument' (singular), 'globals' would be translated to 'Globale Dokumente' (plural)